cargo

```diff
@@ -22,6 +22,7 @@ case "$1" in
     shift
```

Comment:
NIT: We're loosing auto-install in the above case.  Maybe we should check whether `toolchain` is `stable` or `nightly` and source `rust-version.sh` in those cases?

Reply:
True, doesn't look like we use that case anywhere. It feels safer to just remove it, are you opposed to that?

Reply:
Sure, why not!  Past me didn't justify his motivations enough to remind present me. https://github.com/solana-labs/solana/pull/12922 :man_shrugging:

Reply:
Great! I found a [few usages](https://github.com/solana-labs/solana/pull/16360/commits/4969ad58fa8da5504cf41cae965a5e10d501e15d) of it that I replaced with stable/nightly so I think we should be good here, thanks for checking!